include/onnxruntime/core/session/onnxruntime_c_api.h

```diff
@@ -193,6 +193,7 @@ struct OrtCustomOp;
 typedef struct OrtCustomOp OrtCustomOp;
 
 typedef enum OrtAllocatorType {
+  Invalid = -1,
   OrtDeviceAllocator = 0,
```

Comment:
This is the only 'real' change here. The rest is auto formatting by clang including replacing tabs with spaces